src/stores/utils/web3.js

```diff
@@ -98,7 +98,11 @@ export const estimateGas = async (web3, to, gasPrice, from, value, data) =>{
 }
 
 const processWeb3 = async (web3, resolve,  reject) => {
-  const netId = await web3.eth.getChainId()
+  try {
+    const netId = await web3.eth.getChainId()
```

Comment:
Variable `netId` is defined inside the `try` block so it won't be available later for `getNetworkName`. `Type` property of the error is just a type we defined in order to know how to display the error (Alert, specific metamask error window, etc) which are defined on `web3Store.js` as:
```
        rejected: () => this.alertStore.pushError(e.message),
        unlock: () => this.alertStore.pushError(e.message),
        install: () => this.walletInstalled = false
```
 We could add one more for chainId as `chainid: () => this.alertStore.pushError(e.message),` or change the type to a existing one.

Using the seconds options, code could be like:
```
  let netId
  try {
    netId = await web3.eth.getChainId()
  } catch (error) {
    reject({ type: 'unlock', message: 'Wallet does not support getting the Chain ID. Please use another wallet or specify a RPC url of a node that supports eth_chainId call' })
  }
```

Reply:
> Type property of the error is just a type we defined in order to know how to display the error (Alert, specific metamask error window, etc) which are defined on web3Store.js as:

I thought about something like this. Thanks for clarification. I will use 'unlock' type.

Reply:
addressed in https://github.com/poanetwork/bridge-ui/pull/207/commits/47c232a1195f4c1726e5dd5f69dd5e76c5be9e69